pds/contracts/PDS.cdc

```diff
@@ -78,32 +80,32 @@ pub contract PDS{
         }
     }
 
-    pub resource SharedCapabilities {
-        access(self) let withdrawCap: Capability<&{NonFungibleToken.Provider}>
-        access(self) let operatorCap: Capability<&{IPackNFT.IOperator}>
+    access(all) resource SharedCapabilities {
+        access(self) let withdrawCap: Capability<auth(NonFungibleToken.Withdraw) &{NonFungibleToken.Provider}>
```

Comment:
`withdrawCap` & `operatorCap` must now allow access to authorized references to the NFT `Provider` & PackNFT `Operator`, respectively.